src/servers/http_server.cc

```diff
@@ -2325,13 +2342,10 @@ HTTPAPIServer::HandleInfer(
 
   if (err != nullptr) {
     LOG_VERBOSE(1) << "Infer failed: " << TRITONSERVER_ErrorMessage(err);
```

Comment:
You can move this into the helper too, and doesn't need to call TRITONSERVER_ErrorMessage again, you can re-use `message`.